### ISSUE TITLE
Waypoint Crashing

### ISSUE THREAD

**Issue by IBGE-Dev** [THEREALWWEFAN231/Inertia#599]
**Inertia Utility Mod version:3.1** 
**Minecraft version:1.16.1** 
**Operating system:Windows** 
**Optifine/Other mods:Sodium** 

**What's the issue:Editing Waypoints Crashing the game** 

**Crash-report(Would be very useful also please format):
[crash-2020-08-02_01.08.35-client.txt](https://github.com/THEREALWWEFAN231/Inertia/files/5011672/crash-2020-08-02_01.08.35-client.txt)    **

**Comment by THEREALWWEFAN231** [comment]
What did you click, I cant seem to replicate, could you by chance send you Waypoints.json file?

**Comment by IBGE-Dev** [comment]
There you go, sorry for taking so long, i was on work. 
[Waypoints.zip](https://github.com/THEREALWWEFAN231/Inertia/files/5012614/Waypoints.zip)
Uploaded on zip, they don't support json.

**Comment by THEREALWWEFAN231** [comment]
I can not replicate at all.

**Comment by fluse1367** [comment]
Having a simliar issue, but only on multiplayer. It's a nullpointer. Exit code 255. I'm not using other mods. 1.16.3. Inertia 3.1.3.
```java
net.minecraft.class_148: mouseClicked event handler
	at net.minecraft.class_437.method_25412(class_437.java:440) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_312.method_1601(class_312.java:92) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_312.method_22686(class_312.java:162) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_312.method_22684(class_312.java:162) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) ~[lwjgl-glfw-3.2.2.jar:build 10]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar:build 10]
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3174) ~[lwjgl-glfw-3.2.2.jar:build 10]
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:127) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1046) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:656) [intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:215) [intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_265]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_265]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_265]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_265]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192) [fabric-loader-0.9.3+build.207.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140) [fabric-loader-0.9.3+build.207.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26) [fabric-loader-0.9.3+build.207.jar:?]
Caused by: java.lang.NullPointerException
	at inertia.ySjQ.method_25402(Unknown Source) ~[Inertia%20Client%203.1.3%20-%201.16.3.jar:?]
	at net.minecraft.class_312.method_1611(class_312.java:92) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	at net.minecraft.class_437.method_25412(class_437.java:435) ~[intermediary-fabric-loader-0.9.3+build.207-1.16.3.jar:?]
	... 18 more
```

**Comment by THEREALWWEFAN231** [comment]
Fixed in 3.2, when ever that releases.
